src/test/java/org/gridsuite/study/server/SensitivityAnalysisTest.java

```diff
@@ -170,6 +174,7 @@ public void setup() throws IOException {
             @NotNull
             public MockResponse dispatch(RecordedRequest request) {
                 String path = Objects.requireNonNull(request.getPath());
+                System.out.println(path + " shouldNotFind " + difficultyOfFind);
```

Comment:
Volontaire ou petit oubli ?

Reply:
Thanks for catching it. I turn it to LOGGER.debug